autoinstall-schema.json

```diff
@@ -103,6 +103,18 @@
                 }
             ]
         },
+        "kernel-crash-dumps": {
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "type": "boolean"
```

Comment:
Can't this be null too?

Reply:
Ah yes it can, thanks. This came from the original plan and me thinking something like:

```yaml
enabled: null
min-memory: 4G
reserved-memory: 512M
```

Would be kinda weird (custom crashkernel but only if the default says we should enable it I guess?) but really that's fine too.

Will fix shortly.